integration/duckdb/src/lance/duckdb/ml/catalog.cc

```diff
@@ -38,7 +38,8 @@ ModelCatalog* ModelCatalog::Get() {
 }
 
 bool ModelCatalog::Load(const std::string& name, const std::string& uri) {
-  if (models_.contains(name)) {
+  // nvcc doesn't support C++20 yet so can't use std::map::contains()
```

Comment:
would it have impact to including lance codebase?

Reply:
are you thinking we're going to want to merge this with main lance codebase at some point?

Reply:
i was thinking if we provide native reader/writer ext from duckdb extension, similar to parquet extension. 

But we can leave it to later discussion ofc.